canonical-web-and-design/ubuntu.com
# [Image module] /

## Done

Replaced images with image_template module on:
- 400, 404 and 500 pages
  - 404 is easiest to test, and they all use the same image
- homepage takeovers
  - this includes updating the takeover template

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: http://0.0.0.0:8001/jeff-goldblum
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- See that the images on the 404 page are still there (compare to ubuntu.com/jeff-goldblum)
- Visit the homepage, see that the takeover has an image, refresh the page and see that all the takeovers have an image
